README.md

```diff
@@ -67,4 +67,49 @@ C:\> build.cmd nuget SignClientSecret={your secret} SignClientUser={your usernam
 
 This will invoke the `SignClient` and actually execute code signing against your `.nupkg` files prior to NuGet publication.
 
-If one of these two values isn't provided, the code signing stage will skip itself and simply produce unsigned NuGet code packages.
\ No newline at end of file
+If one of these two values isn't provided, the code signing stage will skip itself and simply produce unsigned NuGet code packages.
+
+##Project Description
+
+This project is designed to detect a sudden increase in the TCP port count in a system, and gracefully shutdown the `ActorSystem` when this is observed. 
+
+###Methodology 
```

Comment:
Need to mention the minimum / maximum port count behavior here too. Absolute metrics vs. burst metrics.